crates/store/src/types/application.rs

```diff
@@ -11,6 +11,8 @@ pub struct ApplicationMeta {
     pub blob: key::BlobMeta,
     pub version: Option<Box<str>>,
     pub source: Box<str>,
+    // contract_app_id - Application ID is hash(owner,name) saved in package manager contract
+    pub contract_app_id: Option<Box<str>>,
```

Comment:
We don't want any lock-in to the registry. No allusion to dependence either.

You can use a generic entry called `metadata` as an arbitrary byte slice - `Box<[u8]>`.

And leave it up to clients to visualize that however they deem fit.

Reply:
Updated to metadata name

Reply:
updated to u8